fixups/RegLegacyFixups/RegistryFixups.cpp

```diff
@@ -518,6 +724,622 @@ DECLARE_STRING_FIXUP(RegOpenKeyTransactedImpl, RegOpenKeyTransactedFixup);
 
 
 
+/// <summary>
+/// Method to interpret Regitry Path
+/// </summary>
+/// <param name="regPath"></param>
+/// <returns></returns>
+std::string InterpretRegistryPath(std::string regPath)
```

Comment:
Why have created this new function and not used ReplaceRegistrySyntax?

Reply:
I don't see that you are doing anything different than the purpose of this method. Could you please converge your separate thread and make sure we don't have duplication of code. Thanks

Reply:
As discussed with Rahul, we need to understand more on existing method (maybe existing things are done for a purpose). I have created an issue to track this. 
https://github.com/microsoft/MSIX-PackageSupportFramework/issues/246
Until the issue is closed, I'm not looking to converge these methods.